js/src/rive_advanced.mjs.d.ts

```diff
@@ -42,6 +45,8 @@ interface RiveOptions {
     clipPath(path: RenderPath): void;
     clear(): void;
     flush(): void;
+    translate(x: number, y: number): void;
+    rotate(angle: number): void;
```

Comment:
Curious how translate/rotate methods are related to adding types to the canvas packages. (What does it mean to add types to them?)

Reply:
For projects who consume this runtime, if they're using TypeScript, it's an affordance for them when their project compiles (also is good for documentation purposes). One nice affordance as well is for folks using VSCode, they can have nice things like autocomplete bc it can sense the types of certain objects. so When they do:
```
const renderer = rive.makeRenderer(canvas);
...
renderer.rotate(Math.atan2(x, y));
```
The editor doesn't complain because we've specified `rive.makeRenderer()` returns type `CanvasRenderer` which with these changes, would know `.rotate()` is now a function that can be invoked on `renderer`.

We need to still define translate and rotate on the `skia_renderer.js` in a separate change though, so that's why I mention right now, this will satisfy the `canvas` packages.

Reply:
Got it. Thanks for explaning that!